src/Remote/Data.elm

```diff
@@ -337,3 +337,24 @@ isNotAsked data =
 
         _ ->
             False
+
+
+{-| This helper was projected for reducing switch-case entries when working with collections.
+
+    case
+        entries
+            |> Dict.get id
+            |> RemoteData.reduceMaybe
+    of
+        RemoteData.NotAsked ->
+            Element.text "Request was never prompted"
```

Comment:
Maybe you should also mention that either the request was not prompted or the element was not found (which yes should mean the same, but not necessarily)